addons/survey/wizard/survey_invite.py

```diff
@@ -128,7 +128,7 @@ def _onchange_partner_ids(self):
             if not self.survey_id.users_can_signup:
                 invalid_partners = self.env['res.partner'].search([
                     ('user_ids', '=', False),
-                    ('id', 'in', self.partner_ids.ids)
+                    ('id', 'in', self.partner_ids._origin.ids)
```

Comment:
Let `a` denote an actual record, `b` denote a new record with origin `b1`, and `c` denote a new record without origin.  Then we have:
```python
(a + b + c)._origin = a + b1
(a + b + c).ids = [a.id, b1.id]
```
In other words, `recs.ids == recs._origin.ids`.